samples/http/receiver-gin/README.md

```diff
@@ -18,11 +17,11 @@ Run the app
  go run main.go
 ```
 
-Test a CloudEvent
+Send a CloudEvent
 ```shell
 curl -v \
  -H "Ce-Id: e7d95c20-6eb4-4614-946d-27b0ce41c7ff" \
- -H "Ce-Source: /apis///namespaces/dimitar//clone-build-n4qhgl" \
+ -H "Ce-Source: /apis/namespaces/dimitar/clone-build-n4qhgl" \
```

Comment:
You need to change this in the output below too

Reply:
Source changed in the output as well.